docs/source/aryn_cloud/aryn_partitioning_service.rst

```diff
@@ -0,0 +1,148 @@
+
+.. code:: python
+
+    aryn_api_key = "PUT API KEY HERE"
+
+    ctx = sycamore.init()
+    pdf_docset = context.read.binary(work_dir, binary_format="pdf")
+    partitioned_docset = pdf_docset.partition(ArynPartitioner(aryn_api_key=aryn_api_key))
+
+Alternatively, we can store our Aryn API key at ``~/.aryn/config.yaml`` like so:
+
+.. code:: yaml
+
+    aryn_token: "PUT API KEY HERE"
+
+Which makes our Sycamore script the following:
+
+.. code:: python
+
+    ctx = sycamore.init()
+    pdf_docset = context.read.binary(work_dir, binary_format="pdf")
+    partitioned_docset = pdf_docset.partition(ArynPartitioner())
+
+
+If you are processing a large PDF with OCR, you might benefit from using the ``pages_per_call`` option. This is only available when using the Partition function in Sycamore. This option divides the processing of your document into batches of pages, and you specify the size of each batch.
+
+Using ``curl``
+++++++++++++++
+
+We recommend using the Aryn SDK, but you can also use ``curl`` to access the Aryn Partitioning Service directly.
+
+``curl`` an example document to use with the partitioning service if you do not have one already.
+
+.. code:: bash
+
+    curl http://arxiv.org/pdf/1706.03762 -o document.pdf
+
+Change ``PUT API KEY HERE`` below to your Aryn API key. If you have a different document, change ``@document.pdf`` to ``@/path/to/your/document.pdf`` below.
+
+.. code:: bash
+
+    export ARYN_API_KEY="PUT API KEY HERE"
+    curl -s -N -D headers "https://api.aryn.cloud/v1/document/partition" -H "Authorization: Bearer $ARYN_API_KEY" -F "pdf=@document.pdf" | tee document.json
+
+Your results have been saved to ``document.json``.
+
+.. code:: bash
+
+    cat document.json
+
+Specifying Options
+++++++++++++++++++
+
+We can extract the table structure from our document with the following command.
+
+.. code:: bash
+
+    export ARYN_API_KEY="PUT API KEY HERE"
+    curl -s -N -D headers "https://api.aryn.cloud/v1/document/partition" -H "Authorization: Bearer $ARYN_API_KEY" -F "pdf=@document.pdf" -F 'options={"extract_table_structure": true}' | tee document.json
+
+The available options are listed below:
+
+* ``threshold``: The threshold to use for accepting the models predicted bounding boxes. A lower value will include more objects, but may have overlaps, a higher value will reduce the number of overlaps, but may miss legitimate objects. It defaults to ``0.4``.
+* ``use_ocr``: If ``true``, the partitioner uses OCR to extract text from the PDF. It defaults to ``false``, where the partitioner attempts to directly extract the text from the underlying PDF in the bounding box. It currently uses Tesseract for extraction.
+* ``extract_table_structure``: If ``true``, the partitioner runs a separate table extraction model to extract cells from regions of the document identified as tables.
```

Comment:
"separate table extraction model" is a bit confusing here. Separate from what?

Reply:
Separate from the segmentation/partitioning model.

Reply:
Added context.